.travis.yml

```diff
@@ -15,10 +15,11 @@ matrix:
   # This represents the minimum Rust version supported by Tokio. Updating this
   # should be done in a dedicated PR and cannot be greater than two 0.x
   # releases prior to the current stable.
-  - rust: 1.26.0
+  - rust: 1.28.0
   - rust: stable
   - rust: beta
   - rust: nightly
+    env: ALLOW_FAILURES=true
```

Comment:
Any particular reason why you are using `ALLOW_FAILURES=true` instead of the official [`allow_failures`](https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail) key supported by Travis?

Reply:
The allow_failures section is where you specify what things to allow failures for, but the ALLOW_FAILURES=true part is a way to map which specific thing. Since we have two nightly options -- one basic option + one that runs the async/await stuff -- it had to be customized with ALLOW_FAILURES=true so that they can be differentiated from each other, it seems like?

TravisCI requires you to match the sections in the allow_failures block to how they're written in the matrix block. So, long story short, I added ALLOW_FAILURES=true to the normal nightly build not because it actually adds the ability, but because it makes the whole entry match with the stanza under allow_failures, thus actually picking it up as an allowed failure.

Reply:
Ohhh, I see. `allow_failures` is a matcher, not a separate set of tasks to run. TIL!